crates/flox-rust-sdk/src/models/floxmeta/environment.rs

```diff
@@ -214,6 +221,10 @@ impl<'flox, Git: GitProvider> Environment<'flox, Git, ReadOnly<Git>> {
         Cow::from(&self.system)
     }
 
+    pub fn systematized_name(&self) -> String {
+        format!("{0}.{1}", self.system, self.name)
+    }
```

Comment:
This is identical to the function for project environments. Do you want to factor them out somehow, or leave them separate?